.github/workflows/vercel.yml

```diff
@@ -68,14 +70,15 @@ jobs:
           initial-status: 'in_progress'
           task: deploy:preview
           token: '${{ github.token }}'
+          transient-environment: true
```

Comment:
This change along with `auto-inactive: false` was necessary to prevent previous deployments from being set to Inactive whenever a new deployment occurs.